lang/en/block_massaction.php

```diff
@@ -65,6 +65,9 @@
 $string['noactionsavailable'] = 'You do not have the permissions to execute any of the possible operations this block is providing';
 $string['nomovingtargetselected'] = 'Please select a target section';
 $string['sectionnotexist'] = 'Target section does not exist';
+$string['applicablecourseformats'] = 'Applicable course formats';
+$string['applicablecourseformats_description'] = 'Select for which course formats the Mass Actions block should be available for the users. '
```

Comment:
Select for which course formats the Mass Actions block should be available for the users.
to
Mass Actions block will only be available for the selected course formats.